angular-firebase/src/auth/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import * as firebase from 'firebase';
+import { AngularFirestore } from '@angular/fire/firestore';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.component.html',
+  styleUrls: ['./sign-in.component.css']
+})
+export class SignInComponent implements OnInit {
+  db;
+  constructor(db: AngularFirestore) { this.db = db}
```

Comment:
You could implement dependency injection like this: `private / public angularFirestore: AngularFirestore`
then `this.angularFirestore` is accessible in the component.